src/workstation-gui/app/assets/config/manifest.js

```diff
@@ -0,0 +1,4 @@
+//= link_tree ../images
+//= link_directory ../stylesheets .css
+//= link_tree ../../javascript .js
```

Comment:
do we need this file? frontend-centric file it seems and API is only the backend.

Reply:
Yeah, I'll check its dependency with the rails package